ui/src/components/NodePageMetricsTab.js

```diff
@@ -1,8 +1,436 @@
+  );
+  const workloadPlaneNetworkBandwidthInOperated = operateMetricRawData(
+    nodeStats?.workloadPlaneNetworkBandwidthIn[0]?.values,
+  );
+  const workloadPlaneNetworkBandwidthOutOperated = operateMetricRawData(
+    nodeStats?.workloadPlaneNetworkBandwidthOut[0]?.values,
+  );
+
+  const cpuUsageData = cpuUsageOperated?.map((slot) => {
+    return {
+      date: fromUnixTimestampToDate(slot[0]),
+      y: slot[1],
+    };
+  });
+  const systemLoadData = systemLoadOperated?.map((slot) => {
+    return {
+      date: fromUnixTimestampToDate(slot[0]),
+      y: slot[1],
+    };
+  });
+  const memoryData = memoryOperated?.map((slot) => {
+    return {
+      date: fromUnixTimestampToDate(slot[0]),
+      y: slot[1],
+    };
+  });
+
+  const nodeIOPSWriteData = iopsWriteOperated?.map((slot) => {
+    return {
+      date: fromUnixTimestampToDate(slot[0]),
+      write: slot[1],
+      type: 'write',
+    };
+  });
+  const nodeIOPSReadData = iopsReadOperated?.map((slot) => {
+    return {
+      date: fromUnixTimestampToDate(slot[0]),
+      read: slot[1],
+      type: 'read',
+    };
+  });
+  const nodeControlPlaneNetworkBandwidthInData = controlPlaneNetworkBandwidthInOperated?.map(
+    (slot) => {
+      return {
+        date: fromUnixTimestampToDate(slot[0]),
+        in: slot[1],
+        type: 'in',
+      };
+    },
+  );
+  const nodeControlPlaneNetworkBandwidthOutData = controlPlaneNetworkBandwidthOutOperated?.map(
+    (slot) => {
+      return {
+        date: fromUnixTimestampToDate(slot[0]),
+        out: slot[1],
+        type: 'out',
+      };
+    },
+  );
+
+  const nodeWorkloadPlaneNetworkBandwidthInData = workloadPlaneNetworkBandwidthInOperated?.map(
+    (slot) => {
+      return {
+        date: fromUnixTimestampToDate(slot[0]),
+        in: slot[1],
+        type: 'in',
+      };
+    },
+  );
+  const nodeWorkloadPlaneNetworkBandwidthOutData = workloadPlaneNetworkBandwidthOutOperated?.map(
+    (slot) => {
+      return {
+        date: fromUnixTimestampToDate(slot[0]),
+        out: slot[1],
+        type: 'out',
+      };
+    },
+  );
+
+  // Combine the read/write, in/out into one dataset
+  const iopsData = nodeIOPSWriteData?.concat(nodeIOPSReadData);
+  const controlPlaneNetworkBandwidthData = nodeControlPlaneNetworkBandwidthInData?.concat(
+    nodeControlPlaneNetworkBandwidthOutData,
+  );
+  const workloadPlaneNetworkBandwidthData = nodeWorkloadPlaneNetworkBandwidthInData?.concat(
+    nodeWorkloadPlaneNetworkBandwidthOutData,
+  );
+
+  const xAxis = {
+    field: 'date',
+    type: 'temporal',
+    axis: {
+      // Refer to all the available time format: https://github.com/d3/d3-time-format#locale_format
+      // format: '%m/%d %H:%M',
+      format: '%H:%M', // when the timespan is `Last 24 hours`
+      // Boolean value that determines whether the axis should include ticks.
+      ticks: true,
+      tickCount: 4,
+      labelAngle: -50,
+      labelColor: '#a8b5c1',
```

Comment:
Shouldn't we have all colors in the theme? (same for chart lines)

Reply:
Indeed, the color of the label should come from the `defaultTheme`. 
However, the color of the chart lines are exceptional color and we don't use them anywhere else yet. For me, it makes sense to hardcode the color today.

Reply:
I'm not sure chart lines are "exceptional" in our UIs, since most of our resources will have some form of monitoring metrics. We can hardcode them today, sure, but it'd be a good idea to open a ticket in core-ui and think about it more.

Reply:
Sure! I will open a ticket in core-ui. 
And actually, the colors we use in the graphs is still an open topic we haven't finalized yet. We still need to work with PO & UX.